notebooks/lesson04_intro-to-random-forests.ipynb

```diff
@@ -0,0 +1,1511 @@
+{
```

Comment:
<p>could those be the learning objectives from the last lecture?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>good catch!</p>